### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by kweem**
### Thank you for your contribution! Please go through this small checklist to prepare your pull request to be reviewed as soon as possible :)
* [ ] All authors were added to `_data/authors.yml` with a picture and an up-to-date bio
* [ ] The blog post image is of ratio 500x293
  * Feel free to leave a comment if you don't know how to resize your image and someone will gladly help you out
* [ ] The [**one sentence per line** method](https://raw.githubusercontent.com/brandon-rhodes/blog/master/texts/brandon/2012/one-sentence-per-line.rst) has been adopted
* [ ] A spell check has been done
* [ ] All links open in a new tab
  * Example: `[Amazing tech blog](https://ordina-jworks.github.io){:target="_blank" rel="noopener noreferrer"}`
* [ ] The post's content properly resizes according to the window's size
  * Tip: Just shorten your window's horizontal length to easily test this
  * To have images properly resize use `class="image fit"`
  * To have YouTube videos properly resize, wrap the iframe into a `<div class="responsive-video">`
* [ ] If a table of content is: check whether all links are working correctly

### Any specific remarks for the reviewers or anything you need help with?
\<enter text here>

### Checklist for the reviewers before merging
* [ ] Publishing date of the post has been correctly set
  * This can be done by renaming the filename or by setting the date in the header section of the post

### CODE REVIEW

File: _posts/2021-07-06-Influential-tech-lead.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Be collaborative
+A top-down approach where nobody has a say in the process and where both vision and goals are pushed downwards, is very easy for team members to reject.
+A different and more sustainable approach is to be a collaborative technical team lead.
+This means that you regularly seek out a diversity of opinions and ideas amongst teammates to solve problems.
+Involve your team to create a shared vision, and identify the behaviours necessary to accomplish it.
+It's all about getting the team to think through these difficulties themselves instead of telling them what to do all the time.
+As a result they are more engaged, feel trusted, and are more likely to take ownership of their work.
+Building meaningful relationships is one of the key responsibilities of a technical team lead.
+
```
Comment (bjorndecraemer) [inline _posts/2021-07-06-Influential-tech-lead.md:46]: I would also add something about taking the time on the WHY. Taking the time to get to understand the why of a different approach, why a team member proposes a different approach. Only when that is clear can choices as a team be made.

File: _posts/2021-07-06-Influential-tech-lead.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+As technical team leads we're often pulled in many directions throughout the day. 
+The most common pitfall is that we listen to reply, not to understand.
+It's easy to be a passive listener by multitasking and only listening to the highlights. 
+While it may seem like you don't have time, making the time to really listen to your team can increase your leadership capacity significantly.
+Active listening is the ability to focus completely on your team, understand their message, comprehend the information and respond thoughtfully in a relevant way.
+This also includes not only capturing the message, but paying attention to subtle hints and non-verbal communication such as tone, emphasis, facial expressions and body language.
+
+When your team members know that they will be heard, they are more likely to openly share their ideas and provide honest feedback.
+Knowing how and when to gather knowledge of your team members is important, since you can then synthesize it into a better solution before deciding the course forward.
+
```
Comment (bjorndecraemer) [inline _posts/2021-07-06-Influential-tech-lead.md:58]: A true (servant) leader only speaks 20-30% of the time. The rest should be spent on listening

File: _posts/2021-07-06-Influential-tech-lead.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+So when you're asking the software engineer on your team to write unit tests, the three elements listed above will cross his mind and he will ask himself questions like: "Is it a good thing to write unit tests?", "Is it in my own interest to write unit tests?", "Does everyone write unit tests or is it just me?", "Do I know how to write unit tests and do I have enough information/time to write them?", ...
+Targeting each and every one of these elements is the key to success when you want your colleague to start writing unit tests.
+
+So what can you do to influence the other person's thinking?
+
+- **"Is the behaviour good and is it right?"**: Identify what the other person cares about, what he thinks is good and what he thinks is the right thing to do. For example, your colleague might think it's super important to finish his user story as quickly as possible. The key thing to do here as a technical team lead is not to be manipulative or dishonest, but to genuinely align your goals with his motivations and things he already cares about. Writing unit tests facilitates change, provides documentation, and ultimately saves time.
```
Comment (bjorndecraemer) [inline _posts/2021-07-06-Influential-tech-lead.md:85]: I would not use 'he' here. 
he/she, he or she , 'that person'

**Review by bjorndecraemer — APPROVED** [review]
Looks very good Martin ;)
Do with my comments what you want, you have approvel anyway

File: _posts/2021-07-06-Influential-tech-lead.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+Once you have that vision and goals, being predictable is key.
+By doing so, everybody knows what you're trying to achieve.
+The Don’t-Repeat-Yourself principle, which is a key principle for each software engineer, does not apply when working with people.
+Communicate your vision and goals over and over again: repetition and predictability are key.
+
+## Be collaborative
+A top-down approach where nobody has a say in the process and where both vision and goals are pushed downwards, is very easy for team members to reject.
+A different and more sustainable approach is to be a collaborative technical team lead.
+This means that you regularly seek out a diversity of opinions and ideas amongst teammates to solve problems.
+Involve your team to create a shared vision, and identify the behaviours necessary to accomplish it.
```
[the remark this answers is not in the document]
Comment (kweem) [reply _posts/2021-07-06-Influential-tech-lead.md:42]: ok

File: _posts/2021-07-06-Influential-tech-lead.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+authors: [martin_kwee]
+title: 'Tips for being an influential technical team lead'
+image: /img/2021-07-06-Influential-tech-lead/influence.jpeg
+tags: [leadership, technical leadership, influence]
+category: Leadership
+comments: true
+---
+
+As a software engineer, I really enjoy creating applications and solving complex technical problems.
+But some years ago, there it suddenly was: I was no longer only a "do-er" but an "oversee-er" as well.
```
[the remark this answers is not in the document]
Comment (kweem) [reply _posts/2021-07-06-Influential-tech-lead.md:12]: do-er and oversee-er sound more personal and this part of the blogpost is more about my personal experience

File: _posts/2021-07-06-Influential-tech-lead.md (hunk 1 above)
Comment (kweem) [reply _posts/2021-07-06-Influential-tech-lead.md:46]: Good point, but I think this is also covered more in depth in the explanation of Icek Ajzen's model

File: _posts/2021-07-06-Influential-tech-lead.md (hunk 2 above)
Comment (kweem) [reply _posts/2021-07-06-Influential-tech-lead.md:58]: good addition! :)

File: _posts/2021-07-06-Influential-tech-lead.md (hunk 3 above)
Comment (kweem) [reply _posts/2021-07-06-Influential-tech-lead.md:85]: ok

**Pull request merged**